### PR TITLE
Feat: minimize options to be able to disable the js compression

### PR DESCRIPTION
**Pull request by kopax**
Sometimes `terser` can cause errors or be unwanted for debugging purpose.

I just had all my documentations broken and this will help me behave quickly when minify step is unwanted. 

Related to https://github.com/terser/terser/issues/490

### CODE REVIEW

**Comment by kopax** [comment]
Related to : https://github.com/styleguidist/react-styleguidist/issues/1442
Please merge this and release asap. This is non breaking change and transparent for all your users.

To me it allow to spread the configuration accross all my project at once, I have react-styleguidist in every libraries, even tinies.

All my build were broken suddenly and I am still trying to find the root cause, having this will allow me to restore documentation without terser, elsewhise I need to hack fix everywhere in my CI files.

Thanks in advance,

File: src/scripts/schemas/config.js [hunk 1]
```diff
@@ -135,6 +135,10 @@ module.exports = {
 	logger: {
 		type: 'object',
 	},
+	minify: {
```
Comment (sapegin) [inline src/scripts/schemas/config.js]:
```suggestion
	minimize: {
```

**Comment by kopax** [comment]
hanks to you, is it possible for you to release a version? We are currently stuck with : https://github.com/webpack-contrib/terser-webpack-plugin/issues/173

Would also be nice to have the fix in 9.1.2, version we are currently using because of [our custom rsg-components](https://github.com/bootstrap-styled/rsg-components), newest version have change the anchor behavior, if that is possible...

**Comment by sapegin** [comment]
Want to try to make a release? I've added to the repo, and here are some docs on the process:

https://github.com/styleguidist/react-styleguidist/blob/master/docs/Maintenance.md

**Comment by kopax** [comment]
Thanks,  I have accepted the invitation and I will release after reading all the documentation .

**Comment by kopax** [comment]
@sapegin I was not able to release with the `Feat` tag, as stated in the contribution guide, features are not automatically released, they are grouped in a milestone.

**Comment by sapegin** [comment]
You need to commit a changelog as explained here:

https://github.com/styleguidist/react-styleguidist/blob/master/docs/Maintenance.md#change-logs

**Comment by kopax** [comment]
I've just done it, on my `master` branch, this is the content:

```md
## New features

* minimize options to be able to disable the js compression, sometimes it can cause errors.

To disable it:

```js
module.exports = {
	minimize: false,
};
```

This is how I did it

```
$ npx tamia-changelog
npx: installed 145 in 30.316s
Generating changelog since v9.1.16...
2 commits found.
$ npx tamia-changelog commit
npx: installed 145 in 23.643s
Commiting changelog...
Done.

Don’t forget to push!
```

I don't know why but there is no changelog on your repo, I assumed that the changelog wasn't pushed as a file but as a commit message.

**Comment by sapegin** [comment]
That's correct, but I don't see this commit in the `master`.

**Comment by kopax** [comment]
Should I use `git add -f` then?

**Pull request merged**